gcloud-sdk 0.30.2 breaks the build: missing `concurrency_mode` in `TransactionOptions::ReadWrite` initializers

gcloud-sdk 0.30.2 regenerated the Firestore protos and `transaction_options::ReadWrite` gained a new `concurrency_mode` field. Since this crate's requirement is `gcloud-sdk = "0.30"`, a plain `cargo update` now pulls 0.30.2 and firestore 0.49.0 fails to compile:

```
error[E0063]: missing field `concurrency_mode` in initializer of `gcloud_sdk::google::firestore::v1::transaction_options::ReadWrite`
  --> firestore-0.49.0/src/db/transaction_models.rs:65:29
   |
65 | ...                   gcloud_sdk::google::firestore::v1::transaction_options::ReadWrite {
   |                       ^^^ missing `concurrency_mode`

error[E0063]: missing field `concurrency_mode` in initializer of `gcloud_sdk::google::firestore::v1::transaction_options::ReadWrite`
  --> firestore-0.49.0/src/db/transaction_models.rs:76:29
```

Master is affected too — `src/db/transaction_models.rs` still constructs `ReadWrite` without the field.

Fix would be to set the new field in both initializers (or use `..Default::default()`) and cut a release. Happy to send a PR if that helps. Downstream users can work around it in the meantime with `cargo update -p gcloud-sdk --precise 0.30.1`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ tls-webpki-roots = ["gcloud-sdk/tls-webpki-roots"]
 
 [dependencies]
 tracing = "0.1"
-gcloud-sdk = { version = "0.30", default-features = false, features = ["google-firestore-v1"] }
+gcloud-sdk = { version = "0.30.2", default-features = false, features = ["google-firestore-v1"] }
 hyper = { version = "1" }
 struct-path = "0.2"
 rvstruct = "0.3.2"

--- a/src/db/transaction_models.rs
+++ b/src/db/transaction_models.rs
@@ -64,6 +64,7 @@ impl TryFrom<FirestoreTransactionOptions>
                         gcloud_sdk::google::firestore::v1::transaction_options::Mode::ReadWrite(
                             gcloud_sdk::google::firestore::v1::transaction_options::ReadWrite {
                                 retry_transaction: vec![],
+                                concurrency_mode: gcloud_sdk::google::firestore::v1::transaction_options::ConcurrencyMode::Unspecified as i32,
                             },
                         ),
                     ),
@@ -75,6 +76,7 @@ impl TryFrom<FirestoreTransactionOptions>
                         gcloud_sdk::google::firestore::v1::transaction_options::Mode::ReadWrite(
                             gcloud_sdk::google::firestore::v1::transaction_options::ReadWrite {
                                 retry_transaction: tid,
+                                concurrency_mode: gcloud_sdk::google::firestore::v1::transaction_options::ConcurrencyMode::Unspecified as i32,
                             },
                         ),
                     ),

